day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -60,55 +61,73 @@ Before backing up etcd:
 * Ensure that the {product-title} API service is running.
 * Ensure connectivity with the etcd cluster (port 2379/tcp).
 * Ensure the proper certificates to connect to the etcd cluster.
-* Ensure `go` is installed.
 
 ifeval::["{context}" == "environment-backup"]
-. To ensure the etcd cluster is working, check its health.
-** Run the following command:
+* Ensure that the etcd cluster is working by checking its health:
+** Check the health of the endpoints:
++
+// TODO: is this okay to update to use etcdctl3 (since etcdctl doesn't exist on master node)? Remove ETCDCTL_API=3?
 +
 ----
-# ETCDCTL_API=3 etcdctl --cert="/etc/etcd/peer.crt" \
+# ETCDCTL_API=3 etcdctl3 --cert="/etc/etcd/peer.crt" \
           --key=/etc/etcd/peer.key \
           --cacert="/etc/etcd/ca.crt" \
-          --endpoints="https://*master-0.example.com*:2379,\
-            https://*master-1.example.com*:2379,\
-            https://*master-2.example.com*:2379"
-            endpoint health
+          --endpoints="https://master-0.example.com:2379,https://master-1.example.com:2379,https://master-2.example.com:2379" \// <1>
```

Comment:
@geliu2016 / @hexfusion 

Put all on one line because the command failed when it was on separate lines. Also, removed the asterisks (*), not sure why they were there, in case you know of a reason.